### PR TITLE
[14.0][MIG] helpdesk_mgmt

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration

### CODE REVIEW

**Comment by dalonsod** [comment]
@DarioLodeiros runbot is on error, but I can't see logs. Could you show me the error log?

**Comment by dalonsod** [comment]
@DarioLodeiros @bodedra could you review? Thanks!!

File: helpdesk_mgmt/views/helpdesk_ticket_views.xml [hunk 1]
```diff
@@ -145,9 +145,13 @@
                     </div>
                     <group name="main">
                         <group>
-                            <field name="active" invisible="1" />
+                            <field name="active" attrs="{'invisible': 1}" />
                             <field name="team_id" options='{"always_reload": True}' />
-                            <field name="user_ids" invisible="1" readonly="1" />
+                            <field
+                                name="user_ids"
+                                invisible="1"
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt/views/helpdesk_ticket_views.xml]: attrs="{'invisible': 1} ??

**Review by emagdalenaC2i — APPROVED** [review]
LGTM just a minor note

File: helpdesk_mgmt/views/helpdesk_ticket_views.xml (hunk 1 above)
Comment (pedrobaeza) [reply helpdesk_mgmt/views/helpdesk_ticket_views.xml]: We have to move to attrs only when the condition is dynamic. In this case, it's static and it's correct as is.

Comment (emagdalenaC2i) [reply helpdesk_mgmt/views/helpdesk_ticket_views.xml]: Thanks for the info

Comment (dalonsod) [reply helpdesk_mgmt/views/helpdesk_ticket_views.xml]: You're right, I forgot it, thanks!

Comment (pedrobaeza) [reply helpdesk_mgmt/views/helpdesk_ticket_views.xml]: @dalonsod please revert the change, as is not needed and the syntax is incorrect: attts expect a domain, not a simple value

File: helpdesk_mgmt/data/helpdesk_data.xml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        </record>
+        <record id="helpdesk_ticket_stage_awaiting" model="helpdesk.ticket.stage">
+            <field name="sequence">3</field>
+            <field name="name">Awaiting</field>
+            <field name="unattended">False</field>
+            <field name="closed">False</field>
+            <field name="company_id" />
+        </record>
+        <record id="helpdesk_ticket_stage_done" model="helpdesk.ticket.stage">
+            <field name="sequence">3</field>
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt/data/helpdesk_data.xml]:
```suggestion
            <field name="sequence">4</field>
```

File: helpdesk_mgmt/data/helpdesk_data.xml [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+            <field name="name">Done</field>
+            <field name="unattended">False</field>
+            <field name="closed">True</field>
+            <field name="fold">True</field>
+            <field name="mail_template_id" ref="helpdesk_mgmt.closed_ticket_template" />
+            <field name="company_id" />
+        </record>
+        <record id="helpdesk_ticket_stage_cancelled" model="helpdesk.ticket.stage">
+            <field name="sequence">4</field>
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt/data/helpdesk_data.xml]:
```suggestion
            <field name="sequence">5</field>
```

File: helpdesk_mgmt/views/helpdesk_ticket_views.xml (hunk 1 above)
Comment (dalonsod) [reply helpdesk_mgmt/views/helpdesk_ticket_views.xml]: @pedrobaeza done!

File: helpdesk_mgmt/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
+* `Domatix <https://www.domatix.com>`_:
+
+  * Carlos Mart??nez
+  * Catalin Airimitoaie
+  * ??lvaro L??pez
```
Comment (eLBati) [inline helpdesk_mgmt/readme/CONTRIBUTORS.rst]: Question marks?

Comment (dalonsod) [reply helpdesk_mgmt/readme/CONTRIBUTORS.rst]: Fixed

**Comment by luffah** [comment]
Portal side is a little visually broken, but functionnal.

**Comment by dalonsod** [comment]
@pedrobaeza @eLBati could you review? Thanks!

File: helpdesk_mgmt/i18n/es.po [hunk 5]
```diff
@@ -0,0 +1,1499 @@
...
+"      </data>"
+
+#. module: helpdesk_mgmt
+#: model_terms:ir.ui.view,arch_db:helpdesk_mgmt.portal_helpdesk_ticket_page
+msgid ""
+"<br/>\n"
+"                                    <b>Category:</b>"
+msgstr ""
+"<br/>\n"
+"                          <b>Categor??a:</b>"
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt/i18n/es.po]: There are a few lines like this one

File: helpdesk_mgmt/i18n/es.po [hunk 6]
```diff
@@ -0,0 +1,1499 @@
...
+#: model:ir.model.fields,field_description:helpdesk_mgmt.field_helpdesk_ticket_stage__company_id
+#: model:ir.model.fields,field_description:helpdesk_mgmt.field_helpdesk_ticket_tag__company_id
+#: model:ir.model.fields,field_description:helpdesk_mgmt.field_helpdesk_ticket_team__company_id
+msgid "Company"
+msgstr "Compa????a"
+
+#. module: helpdesk_mgmt
+#: model:ir.ui.menu,name:helpdesk_mgmt.helpdesk_ticket_config_main_menu
+msgid "Configuration"
+msgstr "Configuraci??n"
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt/i18n/es.po]: or this one

**Comment by dalonsod** [comment]
Thanks @emagdalenaC2i , I've experienced some problems with `pre-commit` execution.

**Comment by watthanun** [comment]
Helpdesk -> Configuration -> teams -> create 
can't create teams

**Comment by dalonsod** [comment]
> 
> 
> Helpdesk -> Configuration -> teams -> create
> can't create teams
> 

@watthanun you're right, I'll take a look

**Comment by dalonsod** [comment]
> Helpdesk -> Configuration -> teams -> create
> can't create teams

@watthanun fixed, could you check? Thanks!

**Comment by treylade** [comment]
Could you please merge this PR, since it looks fine? I want to use the module in v14. Thanks!

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Comment by mylbco** [comment]
@dalonsod @pedrobaeza since it's ready to merge, can it be merged please ?

**Pull request merged**